Add initial documentation for accepting COVID-19 pilots. (#682)
opensciencegrid/docs: docs/compute-element/covid-19.md

Job routes are controlled by the `JOB_ROUTER_ENTRIES` configuration variable
in HTCondor-CE (customizations may be placed in `/etc/condor-ce/config.d/`).

For example, consider the following route for a CE submitting to a SLURM batch
system:

```
JOB_ROUTER_ENTRIES @=jre
[
 name = "OSG Jobs";
 GridResource = "batch slurm";
 TargetUniverse = 9;
 queue = "osg";
]
```

To add a new route for COVID-19 pilots, append a new route
prior to the existing one:

```
JOB_ROUTER_ENTRIES @=jre
[
 name = "OSG COVID-19 Jobs";
 GridResource = "batch slurm";
 TargetUniverse = 9;
 queue = "covid19";
 Requirements = (TARGET.IsCOVID19 =?= true);
]
[
 name = "OSG Jobs";
 GridResource = "batch slurm";
 TargetUniverse = 9;
 queue = "osg";
]
@jre
```

To verify jobs are being routed appropriately,
[one may examine pilots](compute-element/troubleshoot-htcondor-ce/#condor_ce_router_q)
with `condor_ce_router_q`.

!!! note "Additional considerations for older CEs"
    Prior to HTCondor 8.7.1, HTCondor-CE had separate rules for handling
    multiple routes; see the
    [job router recipes](/compute-element/job-router-recipes) for more
    details.

Similarly, at a HTCondor site, one can place these jobs into a

> separate accounting group by providing the `set_AcctGroup` attribute:

Reviewer: ```suggestion
separate accounting group by providing the `eval_set_AccountingGroup` attribute:
```

```hl_lines="6"
JOB_ROUTER_ENTRIES @=jre
[
 name = "OSG COVID-19 Jobs";
 GridResource = "batch slurm";
 TargetUniverse = 9;

>  set_AcctGroup = "covid19";

Reviewer: ```suggestion
 eval_set_AccountingGroup = "covid19";
```

Good catch @djw8605 but we should use `eval_set` here because the `JOB_ROUTER_DEFAULTS` has `eval_set_AccountingGroup` already

```hl_lines="2 3 4 5 6 7 8 17 18"
JOB_ROUTER_ENTRIES @=jre
[
 name = "OSG_COVID-19_Jobs";
 GridResource = "batch slurm";
 TargetUniverse = 9;
 queue = "covid19";
 Requirements = (TARGET.IsCOVID19 =?= true);
]
[
 name = "OSG_Jobs";
 GridResource = "batch slurm";
 TargetUniverse = 9;
 queue = "osg";
]
@jre

# Specify the order of the routes
JOB_ROUTER_ROUTE_NAMES = OSG_COVID-19_Jobs, OSG_Jobs
```

To verify jobs are being routed appropriately,
[one may examine pilots](/compute-element/troubleshoot-htcondor-ce/#condor_ce_router_q)
with `condor_ce_router_q`.

!!! note "Additional considerations for older CEs"
    Prior to HTCondor 8.8.7 or 8.9.5, HTCondor-CE had separate rules for handling
    multiple routes; see the
    [job router recipes](/compute-element/job-router-recipes#how-jobs-match-to-job-routes) for more
    details.

Similarly, at a HTCondor site, one can place these jobs into a
separate accounting group by providing the `set_AcctGroup` attribute:

```hl_lines="5 11"
JOB_ROUTER_ENTRIES @=jre
[
 name = "OSG_COVID-19_Jobs";
 TargetUniverse = 5;
 set_AcctGroup = "covid19";
 Requirements = (TARGET.IsCOVID19 =?= true);
]
[
 name = "OSG_Jobs";
 TargetUniverse = 5;

>  set_AcctGroup = "osg";

Reviewer: ```suggestion
 eval_set_AccountingGroup = "osg";
```
